docker-compose.yml

```diff
@@ -157,18 +166,18 @@ services:
     volumes:
       - ./data/files/consumer-files/besluiten:/consumer-files/
     restart: always
-    labels:
-      - "logging=true"
     logging: *default-logging
   search:
     image: semtech/mu-search:0.9.0-beta.7
```

Comment:
this one is still missing the logging label

Reply:
Fixed! That should do it

Reply:
That should do it! :)